aurora/contracts/src/AuroraErc20FastBridge.sol

```diff
@@ -434,15 +427,27 @@ contract AuroraErc20FastBridge is Initializable, UUPSUpgradeable, AccessControlU
      * Effects:
      * - Initiates the unlocking process by making a call to the Fast Bridge contract on Near blockchain.
      */
-    function unlock(uint128 nonce, string calldata proof) external whenNotPaused {
+    function unlockAndWithdraw(uint128 nonce, string calldata proof) external whenNotPaused {
```

Comment:
the comment for the function should be fixed

Reply:
166d191c08dd81100ef9f5296dabc07565693757